### PR TITLE
Types: input & input-amount

### PR DESCRIPTION
**Pull request by jorenbroekema**

### CODE REVIEW

File: packages/form-core/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+export { FocusMixin } from './src/FocusMixin.js';
+export { FormatMixin } from './src/FormatMixin.js';
```
Comment (daKmoR) [inline packages/form-core/index.d.ts:2]: I thought that an index.d.ts is only needed if you reexport things from node_modules?

Comment (jorenbroekema) [reply packages/form-core/index.d.ts:2]: Nope. Always needed. I triple checked 2 weeks ago

Comment (daKmoR) [reply packages/form-core/index.d.ts]: was not needed after all... removed

File: packages/localize/types/LocalizeMixinTypes.d.ts [hunk 2]
```diff
@@ -5,6 +5,29 @@ export interface FormatNumberPart {
   value: string;
 }
 
+// Take the DateTimeFormat and add the missing resolved options as well as optionals
+export declare interface FormatDateOptions extends Intl.DateTimeFormatOptions {
```
Comment (daKmoR) [inline packages/localize/types/LocalizeMixinTypes.d.ts:9]: very nice 👍

**Review by daKmoR — APPROVED** [review]

**Pull request merged**